src/sentry/workflow_engine/models/workflow.py

```diff
@@ -15,6 +17,9 @@ class Workflow(DefaultFieldsModel):
     name = models.CharField(max_length=200)
     organization = FlexibleForeignKey("sentry.Organization")
 
+    # Required as the 'when' condition for the workflow
+    condition_group = FlexibleForeignKey(DataConditionGroup, blank=True, null=True)
```

Comment:
Wondering if we should name it as `when_condition_group` or something

Reply:
we def can, one thing i liked about keeping this generic is that we can match the same "required" condition group with detectors if we don't have `when`. I thought it was kinda nice to have those two terms linked because this will be responding to what the detectors emit.